src/i18n/locales/en.ts

```diff
@@ -123,6 +123,7 @@ export const en = {
     showSelectionOnly: 'Show selection only',
     saveNote: 'Save note',
     noDataToShow: 'No data to show',
+    textAboutSomething: 'Text about something',
```

Comment:
I don't think it's a valid placeholder

Reply:
I took a placeholder from figma, what is better to use instead?